src/listeners/MainListener.js

```diff
@@ -112,4 +112,10 @@ module.exports = class MainListener extends EventListener {
       }
     }
   }
+
+  async onVoiceStateUpdate (oldMember, newMember) {
+    const playerManager = this.playerManager.get(newMember.guild.id)
```

Comment:
`this.playerManager` should have a null-check. (no lavalink connection)
Also, change the variable name to `guildPlayer` please.

Reply:
done